src/Layout/QueryParamContext.tsx

```diff
@@ -513,6 +515,8 @@ const QueryParamProvider: FC = ({ children }) => {
           image: paymentSetup.image,
           ownerAvatar: paymentSetup.ownerAvatar,
         });
+        setIsApproved(paymentSetup.isApproved);
+        setIsTopProject(paymentSetup.isTopProject);
```

Comment:
I see TS warnings here - 

"Argument of type 'boolean | undefined' is not assignable to parameter of type 'SetStateAction<boolean>'.
  Type 'undefined' is not assignable to type 'SetStateAction<boolean>"

You need to narrow the type

e.g.

```
setIsApproved(paymentSetup.isApproved ? true : false);
```